docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,157 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Matrix Field Type, you will need to specify a Content ID, a Content Type, and a Field Type. 
+The types used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput`
+- `{TypeIdentifier}{FieldIdentifier}Row`
+
+The example below shows GraphQL query for a List Content Type (Content Type with a Matrix Field Type added), that has two fields:
```

Comment:
The Content Type would be something like "recipe" or "dish", would it not ? Also, we can't use "type" that way, we should use "items".